src/core/cgroup.c

```diff
@@ -3751,9 +3751,7 @@ int manager_notify_cgroup_empty(Manager *m, const char *cgroup) {
 }
 
 int unit_get_memory_available(Unit *u, uint64_t *ret) {
-        uint64_t unit_current, available = UINT64_MAX;
-        CGroupContext *unit_context;
-        const char *memory_file;
+        uint64_t available = UINT64_MAX;
```

Comment:
it might actually make sense to simply initialize this here to physical_memory()

Reply:
i take that back